src/sync_backend.rs

```diff
@@ -158,7 +169,7 @@ pub enum Error {
     NoneBackend,
 
     #[error("Tarmac was rate-limited trying to upload assets. Try again in a little bit.")]
-    RateLimited,
+    RateLimited(u64),
```

Comment:
Can we name that value? From what I see it's the number of seconds to wait before retrying?
```rust
    RateLimited { wait_seconds: u64 },
```
We could also put that time in the error message instead of saying "a little bit"